### PR TITLE
Add links to VMUI from the query panel

### PR DESCRIPTION
**Pull request by Loori-R**
- #34

### CODE REVIEW

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Review by dmitryk-dk — DISMISSED** [review]
LGTM!

File: src/components/PromLink.tsx [hunk 1]
```diff
@@ -14,83 +14,123 @@
...
+  { title: "Last 180 days", duration: "180d" },
+  { title: "Last 1 year", duration: "1y" },
+  { title: "Yesterday", duration: "1d" },
+  { title: "Today", duration: "1d" },
+].map(o => ({
+  id: o.title.replace(/\s/g, "_").toLocaleLowerCase(),
+  ...o
+}))
+
+const PromLink: FC<Props> = (
```
[the remark this answers is not in the document]
Comment (Loori-R) [reply src/components/PromLink.tsx]: renamed to VmuiLink

**Review by dmitryk-dk — APPROVED** [review]
LGTM!

**Pull request merged**